documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md

```diff
@@ -0,0 +1,258 @@
+
+- Which email marketing provider will we use for email-based communications?
+- How will visitors submit their information to be added to a mailing list for Simpler.Grants.gov communications?
+
+### Definition of done
+<!-- Required -->
+
+#### Must have
+
+- [ ] Basic requirements
+  - [ ] Code is deployed to main & PROD through our CI/CD pipeline
+  - [ ] Services are live in PROD (may be behind feature flag)
+  - [ ] All new services have passed a security review (as needed)
+  - [ ] All new services have completed a 508 compliance review (as needed)
+  - [ ] Data needed for metrics is actively being captured in PROD
+  - [ ] Key architectural decisions made about this deliverable are documented publicly (as needed)
+- [ ] Functional requirements
+  - [ ] The static site has been updated with content approved by internal stakeholders
+  - [ ] The front-end infrastructure can handle site traffic of up to 1,000 concurrent users
+  - [ ] Tickets and/or documentation have been created that describe the infrastructure changes needed to support future site traffic at the following levels:
+    - [ ] 10,000 users
+    - [ ] 100,000 users
+    - [ ] 1 million users
+  - [ ] Engineering staff are notified when the site goes down
+  - [ ] Engineering staff have been trained on a documented incident response plan that describes a set of actions to take when the site goes down
+  - [ ] A set of SLAs have been defined internally for site availability and responsiveness
+- [ ] Communications requirements
+  - [ ] An email has been sent out to the subset of public stakeholders with whom we plan to share the static site
+  - [ ] Key internal stakeholders (e.g. help desk staff, HHS leadership) have been notified when the email is sent out so that they can prepare for questions from public stakeholders
+  - [ ] A longer-term stakeholder engagement plan has been drafted that describes future phases of engagement
+  - [ ] Help Desk is notified and trained for any potential support issues that may come through
+
+#### Nice to have
+
+- [ ] Site visitors can subscribe to mailing list to receive updates about the Simpler.Grants.gov initiative
+- [ ] Internal stakeholders can preview a live version of content changes before they are visible to everyone on simpler.grants.gov
+
+### Proposed metrics for measuring goals/value/definition of done
+<!-- Required -->
+
+1. Number of unique site visitors
+2. Total number of site visits
+3. Total number of visits per page
+4. Total number of visits to outbound links to external resources
+5. Site availability
+6. [Lighthouse score](lighthouse) for the site
+7. Deployment build time
+8. Deployment/hosting costs
+9. (Stretch) Number of visitors who subscribe to the mailing list
+
+### Destination for live updating metrics
+<!-- Required -->
+
+Page on the public wiki. **Note:** This will likely change once we deliver [the public measurement dashboard](https://github.com/HHS/simpler-grants-gov/issues/65)
```

Comment:
we can also put a link to our page on https://analytics.usa.gov/

Reply:
I like that too! I added it as another option in [this commit](https://github.com/HHS/simpler-grants-gov/pull/693/commits/bcf247f3b4986382ba7e7c64e9d411d83842b0ae)